lib/renaissance_web/templates/auction/_auction.html.eex

```diff
@@ -0,0 +1,10 @@
+ <ul class="auction-container">
+  <h4 class="auction-title"><%= @auction.title %></h4>
+  <p class="auction-description"><%= @auction.description %></p>
+  <li>Price: <%= @auction.price %></li>
+  <li>Seller: <%= @auction.seller.email %></li>
+  <li>End Date: <%= RenaissanceWeb.AuctionView.puts_utc_as_local(@auction.end_auction_at) %></li>
```

Comment:
I haven't used phoenix like this, but are there helpers so you don't directly call the views? This seems odd?

Reply:
Turns out you can. I didn't realize, but makes sense.

Reply:
Woo

Reply:
[BTW] [Phoenix: Templates Are Just Functions](http://www.jeramysingleton.com/phoenix-templates-are-just-functions/)